### PR TITLE
Deploy aggregating pushgateway and nginx to reverse proxy the pushgateway

### PR DESCRIPTION
**Pull request by jpopelka**
@lbarcziova @csomh I could use some answers here.

I took the templates from the deployment repo.
The `nginx` there creates a [route to `workers.packit.dev`](https://github.com/packit/deployment/blob/master/openshift/nginx.yml.j2#L95) because [it needs to be publicly accessible by the Prometheus @ PSI](https://github.com/packit/deployment#monitoring-of-packit-service).
But if we are going to deploy this to PSI we don't need a public host if the Prometheus runs in PSI, do we?
(leaving aside that we can't create public route in PSI in the first place)

### CODE REVIEW

**Comment by csomh** [comment]
> But if we are going to deploy this to PSI we don't need a public host if the Prometheus runs in PSI, do we?

Exactly. Though we still going to need a route.

File: deployment/roles/deploy/tasks/deploy-configmaps.yml [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
   k8s:
     namespace: "{{ project }}"
     resource_definition: "{{ item }}"
     host: "{{ host }}"
     api_key: "{{ api_key }}"
     validate_certs: "{{ validate_certs }}"
   with_template:
-    - configmap.yml.j2
+    - cm-env-config.yml.j2
+    - cm-nginx.yml.j2
```
Comment (csomh) [inline deployment/roles/deploy/tasks/deploy-configmaps.yml:14]:
```suggestion
    - env-config-cm.yml.j2
    - nginx-cm.yml.j2
```

Just to have a consistent naming across the board.

File: deployment/check.yml [hunk 2]
```diff
@@ -9,7 +9,10 @@
     deployment: "{{ lookup('env', 'DEPLOYMENT') }}"
     DCs:
       - redis
-      - dist2src-worker
+      - workers
```
Comment (csomh) [inline deployment/check.yml:12]: TBH, I like the singular form better, mainly b/c this is how we have in all of our other projects, and this consistency helps with not having to second guess if this is different or not.

File: deployment/roles/deploy/tasks/deploy-secrets.yml [hunk 3]
```diff
@@ -11,6 +11,6 @@
     validate_certs: "{{ validate_certs }}"
   with_template:
     - secret-sentry.yml.j2
-    - centos-secrets.yml.j2
+    - secret-centos.yml.j2
```
Comment (csomh) [inline deployment/roles/deploy/tasks/deploy-secrets.yml:14]: I wouldn't do this, b/c breaks the consistency in this repo, where the type of a resource is indicated as a suffix to the base filenames.

**Comment by jpopelka** [comment]
recheck

**Comment by jpopelka** [comment]
@csomh PTAL, `dist2src-deployment-tests` are green

**Review by csomh — APPROVED** [review]
:rocket:

**Pull request merged**